pkg/services/authn/authn.go

```diff
@@ -47,12 +51,16 @@ type Request struct {
 	// OrgID will be populated by authn.Service
 	OrgID       int64
 	HTTPRequest *http.Request
+
+	// for use in post auth hooks
+	Resp web.ResponseWriter
 }
 
 const (
 	NamespaceUser           = "user"
 	NamespaceAPIKey         = "api-key"
 	NamespaceServiceAccount = "service-account"
+	UserIDPrefix            = "user:"
```

Comment:
nit: Do we need the prefix? We can use the Namespace for user and the NamespacedID constructor to format the id

Reply:
yeah, my bad :)